wrap.js

```diff
@@ -0,0 +1,55 @@
+// Wrap
+
+// wraps a string by a certain width
+// For example, wrap("I am me", 2) => "I \nam \nme"
+
+// str       : defualt ""    :  input line
+// width     : defualt 75    :  width of column of text
+// seperator : defualt '\n'  :  character that seperates lines
+// cut       : defualt false :  should it start a new line in the middle of a word?
+
+
+module.exports = function wrap(str, width, seperator, cut){
+	str = str || "";
```

Comment:
You should use [`makeStr`](https://github.com/epeli/underscore.string/blob/master/helper/makeString.js) here.